src/modules/subcommands/handleBreakout.ts

```diff
@@ -20,13 +21,23 @@ export const handleBreakout: CommandHandler = async (bot, interaction) => {
     return;
   }
 
+  if (interaction.user.id !== "412611450364887048") {
```

Comment:
```suggestion
  if (!(interaction.member as GuildMember)?.permissions.has("MOVE_MEMBERS")) {
```

This would confirm that the member using it has the correct permissions (i.e. our moderation team)

Reply:
nice!!